precompiles/bank/setup_test.go

```diff
@@ -92,6 +100,7 @@ func (s *PrecompileTestSuite) SetupTest() {
 		Symbol:  "XMPL",
 		Display: "xmpl",
```

Comment:
This could also use the `tokenDenom` constant to avoid problems if this should ever be changed in some places